composer.json

```diff
@@ -27,6 +28,7 @@
         "symfony/phpunit-bridge": "^2.7 || ^3.0"
     },
     "provide": {
+        "ext-mongo": "*",
```

Comment:
With php5.x there is only `ext-mongo` as driver for mongodb
With php7.x there is only `ext-mongodb` as driver for mongodb

`sonata-project/doctrine-mongodb-admin-bundle` use `doctrine/mongodb-odm-bundle` and `doctrine/mongodb-odm-bundle` use `doctrine/mongodb-odm`
But the master version of `doctrine/mongodb-odm` needs the old driver `ext-mongo` so is not compatible with php7 and the new driver `ext-mongodb` 
To use `doctrine/mongodb-odm` with php7, the library needs `alcaeus/mongo-php-adapter` to provide `ext-mongo`, making the command translation from `ext-mongo` to `ext-mongodb`
To enable this replacement, it is needed to specify it in the "provide" config. This is fine in my productions ...
Maybe it will be needed to create branchs for <php7 and >=php7